drivers/storage/port/scsiport/scsi.c

```diff
@@ -718,8 +734,16 @@ SpiProcessCompletedRequest(
     /* Scatter/gather */
     if (Srb->SrbFlags & SRB_FLAGS_SGLIST_FROM_POOL)
     {
-        /* TODO: Implement */
-        ASSERT(FALSE);
+        ExFreePoolWithTag(SrbInfo->ScatterGather, TAG_SCSIPORT);
```

Comment:
Normally the assertion should never be triggered. I'll let you decide whether you think adding it is useful or not.
```suggestion
        ASSERT(SrbInfo->ScatterGather);
        ExFreePoolWithTag(SrbInfo->ScatterGather, TAG_SCSIPORT);
```

Reply:
Thanks, Fixed